Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -259,5 +260,50 @@ protected void CheckMixedRealityInputActions()
                 }
             }
         }
+
+        /// <summary>
+        /// Applies the most relevant settings of DataProviderConfigurations
+        /// </summary>
+        /// <param name="typeName">name of the Type of the Data Provider</param>
+        /// <param name="providerAttribute"> the corresponding attribute</param>
+        /// <param name="providerName"> the target provider Name</param>
+        /// <param name="configurationProfile">the target configuration profile</param>
+        /// <param name="runtimePlatform">the target runtime plaform</param>
+        protected void ApplyDataProviderConfiguration(
```

Comment:
This seems mostly identical to 
`BaseDataProviderServiceInspector .ApplyProviderConfiguration`?

Reply:
Also doesn't seem to be used anywhere in this change?

Reply:
This was part of the change that i picked up from #6133. I can take a look at it.

Reply:
removed.